### ISSUE TITLE
[BUG] Inline plots don't always work

### ISSUE THREAD

**Issue by milan338** [twibiral/obsidian-execute-code#59]
When using R for example,

```R
library(datasets)
data(mtcars)
boxplot(mtcars$mpg, col="green")
```

does not work, throwing the error `could not find function "boxpng"`. I suspect this is because of the implementation in `Magic.ts` where it uses the regex `/plot\(.*\)/g` to capture everything with plot and replaces it with a call to `png`, however like in this example, this doesn't always work. It looks like the inline plots will only work specifically for calls to `plot()`, and Python is similarly limited to only `plt.show()`.

**Comment by zcysxy** [comment]
Similar error when using `ggplot2`:

```log
Error in ggpng("/var/folders/kt/8542mdzd03q6v0r_k8n155kc0000gn/T/temp_1663699433172.png") : 
  could not find function "ggpng"
Execution halted
```

**Comment by twibiral** [comment]
This is fixed by release [0.15.1](https://github.com/twibiral/obsidian-execute-code/releases/tag/0.15.0). The fix should be improved by future updates it only supports normal plots at the moment.
